lib/features/family/features/reflect/bloc/summary_cubit.dart

```diff
@@ -76,4 +76,10 @@ class SummaryCubit extends CommonCubit<SummaryDetails, dynamic> {
       ),
     );
   }
+
+  Future<int> getTotalGamePlays() async =>
+      _reflectAndShareRepository.getTotalGamePlays();
+
+  Future<int> getStoreReviewGameCount() async =>
```

Comment:
At first glance its not clear what this is. I would add a comment to clarify (ie. game count after which to show store review)